server/src/com/cloud/servlet/ConsoleProxyServlet.java

```diff
@@ -414,7 +416,7 @@ private String composeThumbnailUrl(String rootUrl, VirtualMachine vm, HostVO hos
         return sb.toString();
     }
 
-    private String composeConsoleAccessUrl(String rootUrl, VirtualMachine vm, HostVO hostVo) {
+    private String composeConsoleAccessUrl(String rootUrl, VirtualMachine vm, HostVO hostVo, Boolean webSocketRequest) {
```

Comment:
boolean should be good. I thought wrapper might help in some cases, but that's not the case here. I'll change it to boolean